application/play/README.md

```diff
@@ -0,0 +1,41 @@
+<p align="center">
```

Comment:
`application/play/scala/example/public/javascripts/main.js`
`application/play/scala/example/public/stylesheets/main.css`
`application/play/java/example/public/javascripts/main.js`
`application/play/java/example/public/stylesheets/main.css`

Are all empty files. Are they necessary?

Reply:
They were auto-generated by the template. 
Since for our example they are not needed, I removed them. Thanks for noticing. ^-^